webui/react/src/utils/flatRun.ts

```diff
@@ -122,12 +122,11 @@ export const removeColumnTypePrefix = (columnName: V1ColumnType): string => {
   return columnName.replace('COLUMN_TYPE_', '');
 };
 
-/// wanna know why this separator is used? see https://hpe-aiatscale.atlassian.net/browse/ET-785
-export const COLUMN_SEPARATOR = '␟';
-export const METADATA_SEPARATOR = '\u241F' as const; // TODO: unify after merging PR 10052
+/// we want to use special characters as "separators" to prevent user input to match with such when creating the arbitrary metadata
+export const METADATA_SEPARATOR = '\u241F' as const;
```

Comment:
this separator is not just for metadata

```suggestion
export const COLUMN_SEPARATOR = '␟';
```

Reply:
its not fixed here.